data_migration/management/commands/import_v1_data.py

```
@@ -67,26 +81,37 @@ def handle(self, *args, **options):
         self._create_tasks(options["skip_task"])
 
     def _import_data(self, data_type: DATA_TYPE, skip: bool) -> None:
+        start_m2m = self.start_type.endswith("m2m")
+
+        if self.start_type:
```

You do this block of code three times.  
You could have a base class for the jobs and have it raise a CommandError("Invalid start type") rather than an empty return?

---

A base class to have a function to set the start_type etc.